java/src/jmri/implementation/DccConsistManager.java

```diff
@@ -20,12 +21,12 @@ public DccConsistManager() {
     }
 
     @Override
-    public Consist addConsist(DccLocoAddress address) {
+    public Consist addConsist(LocoAddress address) {
         if (consistTable.containsKey(address)) {
             return consistTable.get(address);
         }
         DccConsist consist;
-        consist = new DccConsist(address);
+        consist = new DccConsist((DccLocoAddress) address);
```

Comment:
Before this cast, you need to make sure the address is a DCCLocoAddress.  You should throw an illegal argument exception if it isn't.

Reply:
I have fixed these issues.